connector/subgraph/src/aragon-hooks.ts

```diff
@@ -0,0 +1,28 @@
+import { Address } from '@graphprotocol/graph-ts'
+import { loadAppConfig } from './helpers'
+
+/*
+ * Called when an app proxy is detected.
+ *
+ * Return the name of a data source template if you would like to create it for a given appId.
+ * Return null otherwise.
+ *
+ * The returned name is used to instantiate a template declared in the subgraph manifest file,
+ * which must have the same name.
+ */
+export function getTemplateForApp(appId: string): string | null {
+  if (
+    appId ===
+    '0x16c0b0af27b5e169e5f678055840d7ab2b312519d7700a06554c287619f4b9f9'
```

Comment:
This is `gardens-dependency.open.aragonpm.eth` which can be seen here https://github.com/1Hive/gardens-template/blob/72c95353bdc6e6a7f373b825407f8f3753610bd2/contracts/GardensTemplate.sol#L44
and can be double checked here https://rinkeby.aragon.org/#/0x8494952a4b27ba5ceb70da756af1179e16c27604/apps/installed/gardens-dependency.open.aragonpm.eth

We will later updateit to be relative to the network